shortfin/setup.py

```diff
@@ -324,7 +359,7 @@ def populate_built_package(abs_dir):
 
 setup(
     name="shortfin",
-    version="0.9",
+    version=f"{PACKAGE_VERSION}",
```

Comment:
It makes sense to **not** set this to `0.9` here but just as a note, we still do this here
https://github.com/nod-ai/SHARK-Platform/blob/6c8272042a39860e13ee548967bb1c1fcd404c60/shortfin/CMakeLists.txt#L18

Reply:
Yeah more work will be needed to fully plumb versions (including stable, nightly, and dev) through each part of the projects. I tried to focus on the minimal changes needed to get a nightly release going here. In IREE we have some CMake options: https://github.com/iree-org/iree/blob/6c47f631855ced5f48d4897df11784f81ae24386/CMakeLists.txt#L84-L88 that get set through `setup.py` and other scripts: https://github.com/iree-org/iree/blob/6c47f631855ced5f48d4897df11784f81ae24386/compiler/setup.py#L162-L169